espanol/cocinero/dormir/dormir.md

```diff
@@ -1 +1 @@
-Beatito deperto al cocinero
```

Comment:
Ok never mind
El abr 18, 2016 12:13 PM, "Casey Collaborator" <notifications@github.com>
escribió:

> In espanol/cocinero/dormir/dormir.md
> <https://github.com/udacity/create-your-own-adventure/pull/4256#discussion_r60076335>
> :
>
> > @@ -1 +1 @@
> > -Beatito deperto al cocinero
>
> Unfortunately, pull requests that delete or modify lines are not eligible
> for automatic merging. If you wish to have this pull request auto-merged,
> please update it to only add lines. If you don't think you've modified any
> lines, remember that I consider changing the whitespace (e.g. deleting a
> space at the end of a line) a modification, even though it may be hard to
> tell the difference on GitHub.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/4256/files/056573a2ca999f272800684acc2b590f177d6b6f#r60076335>
>